### PR TITLE
WalletConnect fixes

### PR DESCRIPTION
**Pull request by christianbaroni**
- fixes request icon masking in native activity list
- migrates tx confirmation screen to cool-modals

### CODE REVIEW

File: src/screens/Routes/Routes.ios.js [hunk 1]
```diff
@@ -247,25 +243,27 @@ const routesForBottomSheetStack = {
...
+  },
   [Routes.EXPANDED_ASSET_SHEET]: {
     navigationOptions: {
-      allowsDragToDismiss: true,
-      allowsTapToDismiss: true,
       backgroundOpacity: 0.7,
-      blocksBackgroundTouches: true,
       cornerRadius: 24,
       customStack: true,
-      gestureEnabled: true,
```
Comment (brunobar79) [inline src/screens/Routes/Routes.ios.js:268]: These changes look unrelated to the PR. Just checking that you want to include jt

Comment (christianbaroni) [reply src/screens/Routes/Routes.ios.js:268]: Ah yeah that was intentional, just saw it while I was touching the file

99% sure that doesn’t do anything in the slack sheets, no difference when I tested
